File: lect4/input.java
import java.util.Scanner;
public class input{
 public static Scanner scn=new Scanner(System.in);
 public static void input(int [] a)
 {
     for(int i=0;i<a.length;i++)
     {
         a[i]=scn.nextInt();
     }
 } 
 public static void display(int [] a)
 {
     for(int i=0;i<a.length;i++)
     {
         System.out.print(a[i]+" ");

     }
 }
 public static void main(String [] arg)
 {
     int n=scn.nextInt();
     int [] a=new int[n];
     input(a);
     display(a);
 }

}

File: lect4/input.cpp
#include<iostream>
#include<vector>
using namespace std;
void input(vector<int> &a)
{
    for(int i=0;i<a.size();i++)
    {
        cin>>a[i];
    }
}
void display(vector<int> & a)
{
    for(int i=0;i<a.size();i++)
    {
        cout<<a[i]<<" ";
    }
}
int  main(int arg,char** argv)
{
    int n;
    cin>>n;
    vector<int>a(n,0);
    input(a);
    display(a);
    return 0;
}
